src/pages/en/guides/markdown-content.mdx

```diff
@@ -583,6 +569,58 @@ const { minutesRead } = Astro.props.frontmatter;
 </html>
 ```
 
+### Extending Markdown config from MDX
+
+Astro's MDX integration will extend [your project's existing Markdown configuration](/en/reference/configuration-reference/#markdown-options) by default. To override individual options, you can specify their equivalent in your MDX configuration.
+
+The following example disables GitHub-Flavored Markdown and applies a different set of remark plugins for MDX files:
+
+```ts
+// astro.config.mjs
+import { defineConfig } froAm 'astro/config';
+import mdx from '@astrojs/mdx';
+
+export default defineConfig({
+  markdown: {
+    syntaxHighlight: 'prism',
+    remarkPlugins: [remarkPlugin1],
+    githubFlavoredMarkdown: true,
+  },
+  integrations: [
+    mdx({
+      // `syntaxHighlight` inherited from Markdown
+
+      // Markdown `remarkPlugins` ignored,
+      // only `remarkPlugin2` applied.
+      remarkPlugins: [remarkPlugin2],
+      // `githubFlavoredMarkdown` overridden to `false`
+      githubFlavoredMarkdown: false,
+    })
+  ]
+});
+```
+
+You may also prefer to avoid extending your Markdown config from MDX. For this, you can set [the `extendMarkdownConfig` option](/en/guides/integrations-guide/mdx/#extendmarkdownconfig) (enabled by default) to `false`:
```

Comment:
```suggestion
To avoid extending your Markdown config from MDX, set [the `extendMarkdownConfig` option](/en/guides/integrations-guide/mdx/#extendmarkdownconfig) (enabled by default) to `false`:
```